spec/features/assigned_articles_spec.rb

```diff
@@ -23,6 +23,11 @@
       visit "/courses/#{course.slug}/articles/assigned"
       expect(page).to have_content('Nancy Tuana')
       find('a', text: 'Feedback').click
+      # Use a custom wait time for the loading state
+      Capybara.using_wait_time(10) do
+        expect(page).to have_no_content(I18n.t('courses.feedback_loading'))
+      end
+      expect(page).to have_selector('textarea.feedback-form')
       find('textarea.feedback-form').fill_in with: 'This is a great article!'
```

Comment:
The issue i am working on is that the textarea.feedback-form is not rendered imidiatle, because all the users suggestions for that course has to be loaded which takes a bit longer that the normal wait time of Capybara, so the test fails even before the  textarea.feedback-form  is rendered,

Reply:
The normal wait time for Capybara is pretty long, and we rarely need to wait longer. I think the issue here is that `find('textarea.feedback-form')` does not use the wait time, so it will fail immediately if that element has not rendered when it reaches this line. I suspect that the `expect` you've added above is sufficient, without the `.using_wait_time block.`